sys/abi/src/lib.rs

```diff
@@ -479,6 +487,8 @@ pub enum Kipcnum {
     ReadImageId = 4,
     Reset = 5,
     ReadCaboosePos = 6,
+    GetTaskDumpRegion = 7,
```

Comment:
It would be helpful to add these to `kipc.adoc`

Reply:
Done!